client/src/components/ElementEditor/InlineEditForm.scss

```diff
@@ -22,7 +22,7 @@
   & &__form {
     .form-group .form__field-holder {
       flex: 1 0 100%;
-      max-width: 900px;
+      width: 900px;
```

Comment:
The bootstrap styling changed to using `width` instead of `max-width` so this needed to be changed to match